index.html

```diff
@@ -3,134 +3,401 @@
+        <option value="NJ">New Jersey</option>
+        <option value="NM">New Mexico</option>
+        <option value="NV">Nevada</option>
+        <option value="NY">New York</option>
+        <option value="OH">Ohio</option>
+        <option value="OK">Oklahoma</option>
+        <option value="OR">Oregon</option>
+        <option value="PA">Pennsylvania</option>
+        <option value="RI">Rhode Island</option>
+        <option value="SC">South Carolina</option>
+        <option value="SD">South Dakota</option>
+        <option value="TN">Tennessee</option>
+        <option value="TX">Texas</option>
+        <option value="UT">Utah</option>
+        <option value="VA">Virginia</option>
+        <option value="VT">Vermont</option>
+        <option value="WA">Washington</option>
+        <option value="WI">Wisconsin</option>
+        <option value="WV">West Virginia</option>
+        <option value="WY">Wyoming</option>
+       </select>
     </div>
   </div>
 </div>
 <div class="main section">
-  <div class="container">
+  <div class="col-sm-6">
     <div id="chart"></div>
   </div>
+  <div class="col-sm-6">
+    <div id="chart2"></div>
+  </div>
 </div>
 <script type="text/javascript" src="https://npmcdn.com/d3@3.5.17/d3.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/science@1.9.3/science.v1.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/simple-statistics@2.0.0/dist/simple-statistics.min.js"></script>
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    state: 'all',
+    region: 'alll',
   };
-
-  var svg;
-
+  var svg, x, y, svg2, x2, y2;
+  var margin = { top: 15, right: 15, bottom: 40, left: 80 };
+  var width = 700 - margin.right - margin.left;
+  var height = 500 - margin.top - margin.bottom;
   // FETCH DATA
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
+  
 
-  function setup() {
-    // SVG
+  function setup() {    
 
+    // SVG 1
     svg = d3.select('#chart')
-      .append('svg');
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
-    // SCALES
+    //SVG 2
+     svg2 = d3.select('#chart2')
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+    
+    // SCALES 1
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.PovertyRate; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.rate; }))
+      .range([height, 0])
+      .nice();
+
+    rscale = d3.scale.linear()
```

Comment:
Uh oh.

A linear scale for radius. This'll mean that a doubling of the data variable equals a quadrupling of the area. Alex demoed the solution in class ([documentation](https://github.com/d3/d3-3.x-api-reference/blob/master/Quantitative-Scales.md)].

😬